pkg/arduino_sdi_12/Makefile.dep

```diff
@@ -1,4 +1,7 @@
 FEATURES_REQUIRED += periph_gpio_irq
-USEMODULE += atmega_pcint
+
+ifneq (,$(filter cpu_core_atmega,$(FEATURES_USED)))
+  USEMODULE += atmega_pcint
```

Comment:
How about
```suggestion
  USEMODULE += atmega_pcint
else
  USEMODULE += cpp11-compat
```

then?

Reply:
That looks better, but still the same `libstdcpp` error, as if `ifneq (...) ... else ... endif` didn't work as expected?
I've pushed a change so you can look, maybe I've done a silly mistake?